internal/e2e/ts_auto_deps/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "dependencies": {
+    "@bazel/typescript": "file:../build_bazel_rules_typescript/bazel-bin/internal/npm_package",
```

Comment:
Not sure if this will work on Windows. May have to disable tests that pull in the generated npm_package on Windows.

Reply:
e2e tests not run on Windows